src/Stripe/Services/Subscriptions/StripeSubscriptionService.cs

```diff
@@ -10,40 +11,66 @@ public class StripeSubscriptionService : StripeService
 
         public bool ExpandCustomer { get; set; }
 
+        [Obsolete("Get with customerId is deprecated, use Get without the customerId.")]
         public virtual StripeSubscription Get(string customerId, string subscriptionId, StripeRequestOptions requestOptions = null)
         {
-            var url = string.Format(Urls.Subscriptions + "/{1}", customerId, subscriptionId);
+            return Get(subscriptionId, requestOptions);
+        }
+
+        public virtual StripeSubscription Get(string subscriptionId, StripeRequestOptions requestOptions = null)
+        {
+            var url = string.Format(Urls.Subscriptions + "/{0}", subscriptionId);
 
             return Mapper<StripeSubscription>.MapFromJson(
                 Requestor.GetString(this.ApplyAllParameters(null, url, false),
                 SetupRequestOptions(requestOptions))
             );
         }
 
+        [Obsolete("Create with customerId and planId is deprecated, use Create without the customerId planId by supplying those values in the StripeSubscriptionCreateOptions.")]
         public virtual StripeSubscription Create(string customerId, string planId, StripeSubscriptionCreateOptions createOptions = null, StripeRequestOptions requestOptions = null)
         {
-            var url = string.Format(Urls.Subscriptions, customerId);
-            url = this.ApplyAllParameters(createOptions, url, false);
+            createOptions.CustomerId = customerId;
```

Comment:
you could possibly get a null reference here if they don't pass the create options -  it simply needs a null check first.